### DISCUSSION TITLE
Wazuh dashboard, compliance & FIM

### DISCUSSION THREAD

**Discussion by rpbrunetamilia** [Security-Onion-Solutions/securityonion#9884] [Unsupported Versions]
Hi!

We are considering using Security Onion for our platform SIEM/FIM/etc...

I was wondering, with the announcement that Wazuh will not be integrated with SO onward, what are the solutions, out of the box, that will replace different Wazuh premade filters and compliance dashboards?
 
I.E when looking at a log entry (document) it has these fields which are valuable to us:
rule.gdpr IV_32.2
rule.groups windows,  windows_security
rule.hipaa 164.312.b
rule.id 60137

As for the FIM which we use extensively,  what software is replacing wazuh's ?

Thanks

**Comment by InfosecGoon** [comment]
There is File Integrity Monitoring functionality in the Elastic Agent that should be able to handle that use case.

As for the compliance piece, I'm not sure -- is there a particular compliance standard you're working with, or was this just a general question?
